### PR TITLE
W-12588607-stopwatch-collection-cv

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

File: rpa-builder/6.2/modules/ROOT/pages/toolbox-measurement-points-stopwatch-collection.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
+= Stopwatch Collection
+
+The Stopwatch Collection Action Step enables you to collect all timer values from the Stopwatch Action Steps configure in an Activity Workflow.
```
Comment (ms-tlu) [inline rpa-builder/6.2/modules/ROOT/pages/toolbox-measurement-points-stopwatch-collection.adoc:3]: We should mention here that the Stopwatch Collection contains both Timers from the Stopwatch Action Step AND the General Timer Actions Step

**Review by ms-tlu — CHANGES_REQUESTED** [review]
Hi Cristian,
we also updated the Wizard so the Timer / Reached / Value Label input fields are in the same order, so you might want to update the screenshot with the newest version

File: rpa-builder/6.2/modules/ROOT/pages/toolbox-measurement-points-stopwatch-collection.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+The collection of captured stopwatch values in a JSON list, represented as a String. 
+
+== Wizard 
+
+image:stopwatch-collection-wizard.png[50%, 50%, The Stopwatch Collection Wizard]
+
+* *Settings for JSON labels* 
+** *Timer Label* 
++
+The label for the Stopwatch Name in the output JSON
```
Comment (JennyHajee) [inline rpa-builder/6.2/modules/ROOT/pages/toolbox-measurement-points-stopwatch-collection.adoc:40]:
```suggestion
The label for the Stopwatch Name in the output JSON.

```

File: rpa-builder/6.2/modules/ROOT/pages/toolbox-measurement-points-stopwatch-collection.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+** *Reached Label* 
++
+The label for the Stopwatch Reached State in the output JSON
```
Comment (JennyHajee) [inline rpa-builder/6.2/modules/ROOT/pages/toolbox-measurement-points-stopwatch-collection.adoc:43]:
```suggestion
The label for the Stopwatch Reached State in the output JSON.

```

File: rpa-builder/6.2/modules/ROOT/pages/toolbox-measurement-points-stopwatch-collection.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+** *Value Label* 
++
+The label for the Stopwatch Value in the output JSON
```
Comment (JennyHajee) [inline rpa-builder/6.2/modules/ROOT/pages/toolbox-measurement-points-stopwatch-collection.adoc:46]:
```suggestion
The label for the Stopwatch Value in the output JSON.

```

**Review by JennyHajee — APPROVED** [review]

**Pull request merged**